third_party/parser/cc/lexer.rl

```diff
@@ -1247,60 +1266,64 @@ void lexer::set_state_expr_value() {
   # above.
 
   interp_words := |*
-      interp_code => extend_interp_code;
-      interp_var  => extend_interp_var;
-      e_bs escape => extend_string_escaped;
-      c_space+    => extend_string_space;
-      c_eol       => extend_string_eol;
-      c_any       => extend_string;
+      interp_code       => extend_interp_code;
```

Comment:
Just fixing the indent in those blocks since I added `interp_digit_var`, other actions are unchanged.

Reply:
I removed that commit